explore-analyze/query-filter/languages/esql-rest.md

```diff
@@ -64,22 +64,41 @@ POST /_query?format=txt
 The URL parameter takes precedence over the HTTP headers. If neither is specified then the response is returned in the same format as the request.
 ::::
 
+#### Structured formats
+
+Complete responses with metadata. Useful for automatic parsing.
 
 | `format` | HTTP header | Description |
 | --- | --- | --- |
-| Human readable |
-| `csv` | `text/csv` | [Comma-separated values](https://en.wikipedia.org/wiki/Comma-separated_values) |
+| Structured | | _Complete response with metadata_ |
 | `json` | `application/json` | [JSON](https://www.json.org/) (JavaScript Object Notation) human-readable format |
+| `yaml` | `application/yaml` | [YAML](https://en.wikipedia.org/wiki/YAML) (YAML Ain’t Markup Language) human-readable format |
```

Comment:
```suggestion
| `yaml` | `application/yaml` | [YAML](https://en.wikipedia.org/wiki/YAML) human-readable format |
```

nit: not sure if it's useful to provide the recursive acronym 😄

Reply:
I also don't think it adds much, but we have it in every other format, so I'd say to keep it, even if just for normalization and _flavor_ 👀

Reply:
nothing wrong with a bit of flavor 🫡